layouts/docs/single.html

```diff
@@ -22,6 +22,28 @@
                 {{ if ne .Title "" }}
                     <h1>{{ .Title }}</h1>
                 {{ end }}
+                
+                <!-- Render Glossary Page. -->
```

Comment:
@cnunciato, you were wondering if this should be a regular layout instead of a shortcode if it isn't going to be used anywhere else, but I actually think it should be a shortcode even if it's only used in on place. Otherwise we have to do this if check here (which I am not a fan of doing), or we have to essentially duplicate this `single.html` file and modify it to include the below (also not a huge fan of doing that). Unless you have another idea?

In the meantime, I've added a commit that just uses the shortcode for now.